.buildkite/pipeline.ml.yml

```diff
@@ -348,6 +348,15 @@
     - DATA_PROCESSING_TESTING=1 PYTHON=3.7 ./ci/travis/install-dependencies.sh
     - bazel test --config=ci $(./scripts/bazel_export_options) --build_tests_only python/ray/workflow/...
 
+
+- label: ":slot_machine: Ray ML tests"
+  conditions: ["RAY_CI_ML_AFFECTED"]
```

Comment:
this part is duplicated in https://github.com/ray-project/ray/pull/22926/files

@krfricke @matthewdeng did you sync on this?

Reply:
Thanks for noticing, this happened approx. at the same time. I merged #22926 and will apply the changes in this PR